lnwallet/channel.go

```diff
@@ -1910,11 +1937,18 @@ func (lc *LightningChannel) restorePendingRemoteUpdates(
 			}
 
 		case FeeUpdate:
-			lc.remoteUpdateLog.restoreUpdate(payDesc)
+			if heightSet {
+				payDesc.addCommitHeightRemote = height
+				payDesc.removeCommitHeightRemote = height
+			}
 
-		default:
 			lc.remoteUpdateLog.restoreUpdate(payDesc)
+		default:
+			if heightSet {
+				payDesc.removeCommitHeightRemote = height
```

Comment:
As we haven't yet signed for them, shouldn't this be the local height here? Or this is like anticipating sending these updates with our very next signature so they've effectively been committed here?

Reply:
Yeah so this is on the wrong commit. This needs to be on the pull request that moves the deletion of "remote updates we need to sign" from when we sign a commitment to when we receive a revocation.

Reply:
Moved these changes to #4431